R/pkg/R/generics.R

```diff
@@ -1168,3 +1168,7 @@ setGeneric("kmeans")
 #' @rdname fitted
 #' @export
 setGeneric("fitted")
+
+#' @rdname naiveBayes
+#' @export
+setGeneric("naiveBayes", function(formula, data, ...) { standardGeneric("naiveBayes") })
```

Comment:
Yes, I have a test to ensure that we can use the prefix to call e1071::naiveBayes().